app/repositories/stage-execution-repository.js

```diff
@@ -0,0 +1,40 @@
+const { models } = require('../data')
+/**
+ * Get stage execution by id
+ * @returns stage execution object
+ */
+async function get () {
```

Comment:
the comment says get by id, there is no id param

if what the function does is findAll, it'd be better to call the function findAll or getAll